### PR TITLE
Update the test case name and missing test case for account and logging page

### PR DESCRIPTION
**Pull request by anilvaza-crest**
- Updated and added the test case on account and logging page.
- Added the test case description comment on each test case

### CODE REVIEW

File: tests/ui/test_account.py [hunk 1]
```diff
@@ -216,78 +257,128 @@ def test_account_example_checkbox_checked(self, ucc_smartx_configs):
...
         account = AccountPage(ucc_smartx_configs)
         account.table.delete_row(ACCOUNT_CONFIG["name"])
         assert account.entity.close()
         
     @pytest.mark.account
-    def test_delete_cancel(self, ucc_smartx_configs, add_account):
+    # Verifies cancel functionality at time of delete
+    def test_account_delete_cancel_entity(self, ucc_smartx_configs, add_account):
         account = AccountPage(ucc_smartx_configs)
         account.table.delete_row(ACCOUNT_CONFIG["name"])
```
Comment (rajkarkar-crest) [inline tests/ui/test_account.py:322]: This line will delete the account.
Pass cancel=True in delete_row.

File: tests/ui/test_account.py [hunk 2]
```diff
@@ -216,78 +257,128 @@ def test_account_example_checkbox_checked(self, ucc_smartx_configs):
...
         assert account.entity.cancel()
     
     @pytest.mark.account
-    def test_edit_close(self, ucc_smartx_configs, add_account):
+    # Verifies close functionality at time of edit
+    def test_account_edit_close_entity(self, ucc_smartx_configs, add_account):
         account = AccountPage(ucc_smartx_configs)
         account.table.edit_row(ACCOUNT_CONFIG["name"])
```
Comment (rajkarkar-crest) [inline tests/ui/test_account.py:329]: This line will delete the account.
Pass close=True in delete_row.

File: tests/ui/test_account.py [hunk 3]
```diff
@@ -388,12 +501,14 @@ def test_clone_account_backend_verification(self, ucc_smartx_configs, add_accoun
             }
     
     @pytest.mark.account
-    def test_delete_row_backend_verification(self, ucc_smartx_configs, add_account):
+    # Verifies the account in backend after deleting the account from frontend
+    def test_delete_row_backend_validation(self, ucc_smartx_configs, add_account):
```
Comment (zahrasidhpuri-crest) [inline tests/ui/test_account.py:505]: It should be renamed to test_account_delete_row_backend_validation

File: tests/ui/test_account.py [hunk 4]
```diff
@@ -319,9 +419,20 @@ def test_account_clone_frontend(self, ucc_smartx_configs, add_account):
...
+        account.table.clone_row(ACCOUNT_CONFIG["name"])
+        assert account.entity.name.get_value() == ""
+        assert account.entity.username.get_value() == "TestUser"
+        assert account.entity.multiple_select.get_values() == ['Option One']
+        assert account.entity.auth_key.get_value() == "basic"
     
     @pytest.mark.account
-    def test_add_account_backend_verification(self, ucc_smartx_configs):
+    # Verifies the account in backend after adding account from frontend
+    def test_add_account_backend_validation(self, ucc_smartx_configs):
```
Comment (zahrasidhpuri-crest) [inline tests/ui/test_account.py:435]: It should be renamed to test_account_add_backend_validation

File: tests/ui/test_account.py [hunk 5]
```diff
@@ -343,7 +454,8 @@ def test_add_account_backend_verification(self, ucc_smartx_configs):
             }
 
     @pytest.mark.account
-    def test_edit_account_backend_verification(self, ucc_smartx_configs, add_account):
+    # Verifies the account in backend after editing account from frontend
+    def test_edit_account_backend_validation(self, ucc_smartx_configs, add_account):
```
Comment (zahrasidhpuri-crest) [inline tests/ui/test_account.py:458]: It should be renamed to test_account_edit_backend_validation

File: tests/ui/test_account.py [hunk 6]
```diff
@@ -365,7 +477,8 @@ def test_edit_account_backend_verification(self, ucc_smartx_configs, add_account
             }
     
     @pytest.mark.account
-    def test_clone_account_backend_verification(self, ucc_smartx_configs, add_account):
+    # Verifies the account in backend after cloning account from frontend
+    def test_clone_account_backend_validation(self, ucc_smartx_configs, add_account):
```
Comment (zahrasidhpuri-crest) [inline tests/ui/test_account.py:481]: It should be renamed to test_account_clone_backend_validation

**Review by jayjoshi-crest — APPROVED** [review]

File: tests/ui/test_account.py [hunk 7]
```diff
@@ -146,13 +178,15 @@ def test_multi_select_required_fields(self, ucc_smartx_configs):
...
+    # Verifies required field client id
+    def test_account_required_field_client_id(self, ucc_smartx_configs):
         account = AccountPage(ucc_smartx_configs)
         account.entity.open()
         account.entity.auth_key.select('OAuth 2.0 Authentication')
         assert account.entity.save(expect_error=True) == 'Field Client Id is required'
     
     @pytest.mark.account
+    # Verifies required field client secret
     def test_account_client_secret_required_fields(self, ucc_smartx_configs):
```
Comment (zahrasidhpuri-crest) [inline tests/ui/test_account.py:190]: It has to renamed to test_account_required_field_client_secret

File: tests/ui/test_account.py [hunk 8]
```diff
@@ -163,7 +197,8 @@ def test_account_client_secret_required_fields(self, ucc_smartx_configs):
         assert account.entity.save(expect_error=True) == 'Field Client Secret is required'
 
     @pytest.mark.account
-    def test_account_name_start_with_number_required_fields(self, ucc_smartx_configs):
+    # Verifies whether adding special characters, number in starting of name field displays validation error
+    def test_account_valid_account_name(self, ucc_smartx_configs):
```
Comment (zahrasidhpuri-crest) [inline tests/ui/test_account.py:201]: It has to be renamed to test_account_valid_input_name

File: tests/ui/test_account.py [hunk 9]
```diff
@@ -181,33 +217,38 @@ def test_account_name_length_validation(self, ucc_smartx_configs):
...
+    # Verifies default value of auth type
+    def test_account_default_value_auth_type(self, ucc_smartx_configs):
         account = AccountPage(ucc_smartx_configs)
         account.entity.open()
         assert account.entity.auth_key.get_value() == "basic"
 
     @pytest.mark.account
-    def test_account_auth_list(self, ucc_smartx_configs):
+    # Verifies auth type list dropdown
+    def test_account_list_uth_type(self, ucc_smartx_configs):
```
Comment (zahrasidhpuri-crest) [inline tests/ui/test_account.py:244]: Typo, uth to auth

File: tests/ui/test_account.py (hunk 8 above)
Comment (anilvaza-crest) [reply tests/ui/test_account.py:201]: we are validating account name field in this test case so it should be valid test case name.

File: tests/ui/test_account.py (hunk 1 above)
Comment (anilvaza-crest) [reply tests/ui/test_account.py:322]: Done

File: tests/ui/test_account.py (hunk 3 above)
Comment (anilvaza-crest) [reply tests/ui/test_account.py:505]: Done

File: tests/ui/test_account.py (hunk 4 above)
Comment (anilvaza-crest) [reply tests/ui/test_account.py:435]: Done

File: tests/ui/test_account.py (hunk 5 above)
Comment (anilvaza-crest) [reply tests/ui/test_account.py:458]: Done

File: tests/ui/test_account.py (hunk 6 above)
Comment (anilvaza-crest) [reply tests/ui/test_account.py:481]: Done

File: tests/ui/test_account.py (hunk 7 above)
Comment (anilvaza-crest) [reply tests/ui/test_account.py:190]: Done

File: tests/ui/test_account.py (hunk 9 above)
Comment (anilvaza-crest) [reply tests/ui/test_account.py:244]: Done

**Review by zahrasidhpuri-crest — APPROVED** [review]

**Review by rajkarkar-crest — APPROVED** [review]

**Pull request merged**